docs/installation_configuration_guide/Installation.md

```diff
@@ -0,0 +1,174 @@
+# Installation and configuration guide
+
+Nussknacker relies on several open source components like Flink or Kafka, 
+which need to be installed together with Nussknacker. This document focuses on the configuration of Nussknacker and its integrations with those components; pls refer to Kafka, Flink, ... , 
+documentation for details on how to configure them for optimal performance .
+      
+# Installation
+
+## Docker based installation
+                    
+Nussknacker is available at [Docker hub](https://hub.docker.com/r/touk/nussknacker/). 
+You can check an example usage with docker-compose at [Nussknacker Quickstart Repository](https://github.com/TouK/nussknacker-quickstart).
+
+### Base Image
+
+As a base image we uses `openjdk:11-jdk`. See [Open JDK's Docker hub](https://hub.docker.com/_/openjdk) for more details.
+
+### Container configuration
+
+For basic usage, most things can be configured using environment variables. In other cases, can be mounted volume with own configuration file. 
+See "Configuration" section for more details. NUSSKNACKER_DIR is pointing to /opt/nussknacker.
+
+### Kubernetes - Helm chart
+
+We provide [Helm chart](https://artifacthub.io/packages/helm/touk/nussknacker) with basic Nussknacker setup, including:
+- Flink
+- Kafka
+- Grafana + InfluxDB
+          
+Please note that Flink, Kafka are installed in basic configuration - for serious production deployments you probably want to 
+customize those to meet your needs. 
+
+## Binary package installation
+   
+Released versions are available at [GitHub](https://github.com/TouK/nussknacker/releases)
+
+### Prerequisites
+
+We assume that `java` (recommended version is JDK 11) is on path. 
+
+Please note that default environment variable configuration assumes that Flink, InfluxDB, Kafka
+and Schema registry are running on `localhost` with their default ports configured. 
+See [environment variables](#environment-variables) section for the details.
+Also, `GRAFANA_URL` is set to `/grafana`, which assumes that reverse proxy like [NGINX](https://github.com/TouK/nussknacker-quickstart/tree/main/nginx) is used
+to access both Designer and Grafana. For other setups you should change this value to absolute Grafana URL.
```

Comment:
"but then you should also configure CORS"? Tylko, że chyba nie mamy możliwości skonfigurowania "wybiórczo" CORS, żeby obejmował tylko ten url, prawda?